st-embroider/2023/2023-02-07.md

```diff
@@ -0,0 +1,117 @@
+- CM: clarify what EF means by "things are close" and if AC needs this in days weeks or months
+- EF: what I want for big apps is for them to have a parallel build that is kept green and then it's just a flag
+- and we are keeping track of what is breaking etc
+- you can also keep production building with classic for a long time
+- AC: right now that's where we are. we build on every push to master
+- right now building is working, but test environment is failing
+- EF: sounds like we're aligned 
+- we never want a painfully risky swich over process. we really only want it to be a flag
+- 
+
+### reliablyResolvable !.. on main. Need some guidance on where to debug next <@mansona>
+
+- sorry i was talking and couldn't take notes
+- I have a way forward
+
+### move from zoom to discord <@void_mAlex>
+
+- general consensus to do it
+- CM: I will get one created and let you know
+
+### path towards changing index.html from output to input semantics <@simonihmig>
+
+- SI we've talked about this a few times and I want some guidance from a 10k foot view
+- EF: there is probably a transition implementation that we could do
+- we could make an addon or we could just write it as an RFC
+- then you could be empowered to just make the changes to ember-cli 
+- SI: does this need changes in ember-cli? or can this be a proof of concept on the embroider side?
+- EF: we can do this on the embroider side behind a flag
+- we should call it experimental until we figure out how to do it in classic builds
+- we could also decide which problems are in scope here
+  - files outside of app not addressible
+  - config format
+  - etc. other things that are under the bucket of "v2 app rfc"
+- A: we could do it with a new html file? icantbelieveitsnot.html or somethign
+- we can pass the file name as a flag
+- EF: index.html is important though
+- in embroider you can have any number of html files at lots of different folders
+- i think this needs to be a yes or no flag, you can't have multiple different files
+- A: does this mean we need the v2 app rfc
+- EF: that's what I'm thinking. 
+- CM: it sounds like we need the experiment regardless on the decision on how big this needs to be before landing
+- that way we can decide after how big it will be
+- EF & SI: yes that sounds good
+- CM: added to the roadmap
+- SI: this has considerations for app.css too, do we keep the legacy pipeline
+- EF: we could have a specific import from `embroider-compat/css-pipeline` or something
+- we also have to think about how public works. we would need to not special case public any more, or if it does then you wouldn't use that for things like favicons
+- 
+
+### release blockers <@nullvoxpopuli>
+
+- the resolving pr could be a blocker
+- I'm doing red green development but that won't catch everything (see CM's problems)
+- (discussion about backporting)
+- general consensus is that we can just wait to release, there are a lot of benefits to wait for the work that is in progress
```

Comment:
we added that, after you left, we need to add more tests / figure out why the existing test suite has been a smidge insufficient 😅

Reply:
added 👍